.pipelines/templates/template-job-deploy-azure-env.yml

```diff
@@ -28,20 +28,7 @@ jobs:
       runOnce:
         deploy:
           steps:
-          - task: DownloadBuildArtifacts@0
-            inputs:
-              buildType: specific
-              project: ${{ parameters.vsoProjectID }}
-              pipeline: ${{ parameters.vsoConfigPipelineID }}
```

Comment:
Remove from params list. 

```
parameters:
  aroVersionStorageAccount: ''
  azureDevOpsE2EJSONSPN: ''
  azureDevOpsJSONSPN: ''
  billingE2EPipelineName: ''
  billingE2EBranchName: ''
  configFileName: ''
  e2eSubscription: ''
  environment: ''
  locations: []
  rpMode: ''
  vsoProjectID: ''
  vsoConfigPipelineID: ''
  vsoConfigBuildID: ''
  vsoDeployerPipelineID: ''
  vsoDeployerBuildID: ''
  ```

Reply:
I am aware of it, it will be removed once all comments are present. I left it intentionally.